<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NDS Graphic File Generator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>311</width>
      <height>221</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_ncg">
     <attribute name="title">
      <string>NCG</string>
     </attribute>
     <widget class="QSpinBox" name="ncg_width_sb">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>512</number>
      </property>
      <property name="value">
       <number>256</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Size:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>x</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="ncg_height_sb">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>512</number>
      </property>
      <property name="value">
       <number>192</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="ncg_progressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>271</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="ncg_saveas_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>271</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate And Save As...</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="ncg_is4bpp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>4bpp Format</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="ncg_lz77_cb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>LZ77 Compress</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="ncgr_cb">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>NCGR</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_ncl">
     <attribute name="title">
      <string>NCL</string>
     </attribute>
     <widget class="QCheckBox" name="ncl_is_extended_cb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>256 Colors</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="ncl_progressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>271</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="ncl_pal_n_sb">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>16</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>55</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Palettes:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ncl_saveas_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>271</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate And Save As...</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="ncl_lz77_cb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>LZ77 Compress</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="nclr_cb">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>NCLR</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>NSC</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Size:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>x</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="nsc_width_sb">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>8</number>
      </property>
      <property name="maximum">
       <number>512</number>
      </property>
      <property name="value">
       <number>224</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="nsc_height_sb">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>8</number>
      </property>
      <property name="maximum">
       <number>512</number>
      </property>
      <property name="value">
       <number>224</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="nsc_progressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>271</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="nsc_saveas_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>271</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate And Save As...</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="nsc_nsmbem_cb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Fixed screen width mode (NSMBe)</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="nsc_lz77_cb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>LZ77 Compress</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_nsbtx">
     <attribute name="title">
      <string>NSBTX</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Maybe in the future...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_nsmbd">
     <attribute name="title">
      <string>NSBMD</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Maybe in the future...</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
